### PR TITLE
Tidying up and leaving only dind build

### PR DESCRIPTION
**Pull request by hatemosphere**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -14,26 +17,11 @@ jobs:
...
     runs-on: ubuntu-20.04
-    strategy:
-      matrix:
-        image: ["base","dind"]
     steps:
       - uses: actions/checkout@v2
       - uses: hadolint/hadolint-action@v1.6.0
         with:
           failure-threshold: warning
-      - name: Map runtime environment data
```
Comment (Deniallugo) [inline .github/workflows/build.yml:25]: Are you sure about it? 
maybe it would be useful to rename tag based on file?

File: Dockerfile [hunk 2]
```diff
@@ -1,19 +1,21 @@
-ARG VIRTUAL_ENV_INSTALLS="rust"
-ARG VIRTUAL_ENV_PACKAGES="pkg-config build-essential axel postgresql-client libpq-dev libssl-dev cmake"
-FROM dysnix/github-actions-runner:gcloud
+### Not used anymore, DinD + Docker'ized actions is our way
```
Comment (Deniallugo) [inline Dockerfile:1]: Let's delete it. 
Git remember everything =)

File: .github/workflows/build.yml (hunk 1 above)
Comment (hatemosphere) [reply .github/workflows/build.yml:25]: @Deniallugo i would not overcomplicate this until there's a real plan to add more than one image here :)

Comment (Deniallugo) [reply .github/workflows/build.yml:25]: Ok, just to make sure that we will push latest image

Comment (hatemosphere) [reply .github/workflows/build.yml:25]: Let's see what's going to be pushed 🗡️

File: CODEOWNERS [hunk 3]
```diff
@@ -1,3 +1,3 @@
 # https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners
 
-* @adrianchifor @dennybaa @killmeplz
+* @dennybaa @Deniallugo
```
[the remark this answers is not in the document]
Comment (hatemosphere) [reply CODEOWNERS]: done

**Review by Deniallugo — APPROVED** [review]

**Review by dennybaa — APPROVED** [review]

**Pull request merged**